cg/store/crud/read.py

```diff
@@ -604,7 +625,9 @@ def get_pools_to_render(
                     or set(self.get_pools_by_order_enquiry(order_enquiry=enquiry))
                 )
             )
-        return pools
+        total: int = len(pools)
```

Comment:
Im not super keen on this but refactoring this function (and evaluating if it actually does what we want) seems out of scope to me